### ISSUE TITLE
Mode button cleanup

### ISSUE THREAD

**Issue by pfriedel** [sheaivey/rx5808-pro-diversity#133]
It seems like either the settings screen should be merged into the slide-in popup screen (auto vs manual, channel vs hz, add band scan vs tuner and "settings!") or make the slide in menu as immediate as possible on mode press so it isn't quite so fussy about timing.  On the other hand I'm using it as a base station and not in goggles, so I'm less concerned about inadvertent button presses while I'm blindly groping for the toggle in the module bay.

(I see a (disused?) #define for that already (BUTTON_WAIT_FOR_MENU), so this might be on your development radar.)

**Comment by Knifa** [comment]
Yes, I had a think about this. Putting everything in the side menu (including switching modes, etc) might make for a neat UI, though it's difficult to fit anything in other than icons (and maybe a slug of text). I'm keen to try it out.

I made the menu reachable by a hold rather than single press because I wanted to free up single press for other uses (e.g., selecting things in the menu itself, other things in future like favourites, etc), but I do admit it's a little awkward.
